libnavigation-ui/src/main/java/com/mapbox/navigation/ui/summary/list/InstructionListPresenter.java

```diff
@@ -115,12 +115,16 @@ private void addBannerInstructions(RouteProgress routeProgress) {
     if (isNewLeg(routeProgress)) {
       instructions = new ArrayList<>();
       currentLeg = routeProgress.currentLegProgress().routeLeg();
-      drivingSide = routeProgress.currentLegProgress().currentStepProgress().step().drivingSide();
-      List<LegStep> steps = currentLeg.steps();
-      for (LegStep step : steps) {
-        List<BannerInstructions> bannerInstructions = step.bannerInstructions();
-        if (bannerInstructions != null && !bannerInstructions.isEmpty()) {
-          instructions.addAll(bannerInstructions);
+      if (routeProgress.currentLegProgress().currentStepProgress().step() != null) {
```

Comment:
Same as above re: wondering why we haven't run into this NPE before 🤔

Reply:
In the core SDK, RouteProgress and every object inside it @Nullable. This was not the case earlier. Plus we are trying to start navigation without a route which was not the case earlier.